Makefile

```diff
@@ -46,6 +46,7 @@ appstore:
 	--exclude=phpunit*xml \
 	--exclude=tests \
 	--exclude=vendor/bin \
+	--exclude=vendor/yubico/u2flib-server/examples \
```

Comment:
how about we do both, just to be sure not to package it in case I do a local composer install? :-)